mongo_connector/__init__.py

```diff
@@ -11,7 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import logging
+
 from pkgutil import extend_path
 __path__ = extend_path(__path__, __name__)
 
-__version__ = '2.5.0.dev0'
+version_info = (2, 5, 0, 'dev0')
+version = '.'.join(str(v) for v in version_info)
+__version__ = version
```

Comment:
Can you not still import a `__version__` from the `mongo_connector` module? I just tried installing mongo-connector and the elastic-doc-manager side-by-side, and it looks like this still works. This could still be a problem if some doc manager tries to put its own version in `mongo_connector/__init__.py`, but doc manager authors shouldn't do that or need to.

I don't think we need to add another way to import doc managers. We already have two: use the `extend_path` trick or use the `docManagerClassPath` option in the config file (which circumvents this issue). If we stuck `__version__` in the doc manager module itself, instead of in `__init__.py`, we should be able to import the version from the doc manager without worrying about anything that might be in `mongo_connector/doc_managers/__init__.py`. If we can't import `__version__` reliably from mongo_connector, then maybe we need to put it back in constants.py.